src/test/scala/org/constellation/infrastructure/redownload/RedownloadPeriodicCheckTest.scala

```diff
@@ -21,9 +21,9 @@ class RedownloadPeriodicCheckTest
   before {
     dao = TestHelpers.prepareMockedDAO()
 
-    dao.redownloadService.fetchPeersProposals() shouldReturnF Unit
-    dao.redownloadService.recalculateMajoritySnapshot() shouldReturnF Unit
-    dao.redownloadService.checkForAlignmentWithMajoritySnapshot() shouldReturnF Unit
+    dao.redownloadService.fetchAndSetPeersProposals() shouldReturnF Map()
```

Comment:
why fetchAnd**Set** should behave as a getter?

Reply:
I gave peersProposals a return value and meant to do so with ownSnapshots (but looks like I forgot) to help with testing and remove getOwnSnapshots(). I'm just gonna merge into one map then change this all